### PR TITLE
AuthViewController: SignUp by default

### PR DESCRIPTION
**Pull request by jleandroperez**
### Fix
In this PR we're adjusting the Authentication UI, so that the app starts off with the SignUp UI.

cc @eshurakov YO!!

### Test
1. Fresh install Simplenote

- [x] Verify there is no animation at launch
- [x] Verify the Authentication UI is, at launch, in SignUp mode
- [x] Verify that clicking over the **Log In** button switches the UI to Login 

### Release
These changes do not require release notes.

### CODE REVIEW

**Comment by jleandroperez** [comment]
cc @eshurakov Ready for review sir, thanks in advance!!

File: Simplenote/AuthViewController+Swift.swift [hunk 1]
```diff
@@ -79,37 +74,55 @@ extension AuthViewController {
         switchActionButton.title   = switchText.uppercased()
     }
 
-    /// Displays / Hides components, based on the ViewController Mode (SignUp / LogIn)
+    /// Makes sure unused components (in the current mode) are effectively disabled
     ///
-    func refreshVisibleComponentsWithAnimation() {
-        let mustHide = !signingIn
+    func refreshEnabledComponents(signingIn: Bool) {
```
Comment (eshurakov) [inline Simplenote/AuthViewController+Swift.swift]: `signingIn` is a property on `AuthViewController`, do we really need to pass it around? Will it happen that we will pass a value here that won't match `self.signingIn`? 🤔

**Review by eshurakov — APPROVED** [review]
Added a small question / comment.
Other than that, looks good! 
:shipit:

File: Simplenote/AuthViewController+Swift.swift (hunk 1 above)
Comment (jleandroperez) [reply Simplenote/AuthViewController+Swift.swift]: Too much Erlang. You're right, it makes no sense to actually pass it around!

**Comment by jleandroperez** [comment]
Thank you Evgeny!!

**Pull request merged**